java/managed-vms/README.md

```diff
@@ -63,38 +77,35 @@ This app is a [Jetty](http://www.eclipse.org/jetty/) based [Servlet](http://www.
 
  `create 'gae-hello', 'visits'`<br />
```

Comment:
Perhaps the beginning of the README could mention that the app uses two tables, which must be named `gae-hello` and `from-json`, to store its data. (Right now the reader has to infer this.)